objectPerformance/src/plotter.py

```diff
@@ -280,12 +376,13 @@ def _compute_scalings_tanh(self,
                                scalings,
                                scaling_pct):
         bins = turnon_collection.bins
+        bins = 0.5*(bins[1:] + bins[:-1])
         threshold = turnon_collection.threshold
 
         for obj in turnon_collection.hists:
             if obj == "ref":
                 continue
-            efficiency, _ = turnon_collection.get_efficiency(obj)
```

Comment:
the `_` is a way of dumping the value that is returned from the function, since it is not used downstream. Creating variables that are not used is generally discouraged.